UITests/Tests/AxeScanAllTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestPlatform;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.IO;
+using System.Collections.ObjectModel;
+using System.Collections;
+using System.Collections.Generic;
+using System.Xml;
+using System.Reflection;
+
+namespace UITests.Tests
+{
+    [TestClass]
+    public class AxeScanAll : TestBase
+    {       
+        public static readonly string xmlUri = "WinUIGalleryTestData.xml";
+        public static new WindowsDriver<WindowsElement> Session => SessionManager.Session;
+
+        private static IEnumerable<object[]> TestData()
+        {
+            var testCases = new List<object[]>();
```

Comment:
Since you are doing your own parsing of the data, would it be simpler to directly parse the ControlInfoData.json file instead of having a separate script to convert ControlInfoData.json into WinUIGalleryTestData.xml and then parsing that xml file?

For the Lifted Xaml repo, I needed to convert from json into an xml format that taef understood. But since you re doing your own parsing here, you don't have to use xml at all.

Reply:
I believe it's easier to parse the .xml file instead of the .json. The json file has quite a lot of info that will need to be parsed, but is not relevant to the tests. This method adds an extra file, but results in cleaner test code.

Reply:
What about using the same parsing code from the WinUI 3 Gallery? Since we are using the same .NET SDK, we could move them in a shared project so both projects can reference the same data and parsing.

Reply:
You are already parsing the json file. You are just doing it in powershell as part of the build. You can move that same logic out of powershell and into the test code.

Reply:
@kmahone Done.
@chingucoding That can definitely be a future refactor, but too big of a change for this PR in my opinion.